hathor/verification/merge_mined_block_verifier.py

```diff
@@ -12,8 +12,15 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+from hathor.transaction import MergeMinedBlock
 from hathor.verification.block_verifier import BlockVerifier
 
 
 class MergeMinedBlockVerifier(BlockVerifier):
     __slots__ = ()
+
+    @staticmethod
```

Comment:
Why is it a static method?

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/797/commits/105e0598363e0125e9f6112ec72b213889790b61

Reply:
Thanks!